.github/workflows/image-pr.yml

```diff
@@ -68,7 +68,7 @@ jobs:
           - build-type: 'sycl_f16'
             platforms: 'linux/amd64'
             tag-latest: 'false'
-            base-image: "intel/oneapi-basekit:2024.0.1-devel-ubuntu22.04"
+            base-image: "intel/oneapi-basekit:2024.1.0-devel-ubuntu22.04"
```

Comment:
I'll have to give a test at this, but I don't see anything wrong in bumping the image

Reply:
Very curious to know how it works on Arc.
Still some issue to sort with Llama3 while Phi3-128k is working like a charm.